app/src/main/java/com/android/voyageur/model/trip/TripsViewModel.kt

```diff
@@ -566,7 +584,36 @@ open class TripsViewModel(
             )
 
     tripInviteRepository.createTripInvite(
-        req = tripInvite, onSuccess = onSuccess, onFailure = onFailure)
+        req = tripInvite,
+        onSuccess = {
+          // Update trip invites state
+          _tripInvites.value += tripInvite
+          Log.d("TripsViewModel", "Created invite: ${tripInvite.tripId}")
+          Log.d("UserDropdown", "Trip Invites: $tripInvites")
+          onSuccess()
+        },
+        onFailure = { e ->
+          Log.e("TripsViewModel", "Failed to create invite: $e")
+          onFailure(e)
+        })
+  }
+
+  fun getSentTripInvites() {
+    val userId = firebaseAuth.uid.orEmpty()
+    if (userId.isEmpty()) return
+
+    sentTripInvitesListener?.remove()
+
+    sentTripInvitesListener =
+        tripInviteRepository.listenToSentTripInvites(
+            userId = userId,
+            onSuccess = { invites ->
+              _sentTripInvites.value = invites
+              Log.d("TripsViewModel", "Fetched sent trip invites: ${invites.size}")
```

Comment:
I think you can delete this log, since I believe either was for debugging purposes.

Reply:
Nice catch, I'll delete it! Thanks.